stdlib/public/Platform/wasi-libc.modulemap

```diff
@@ -19,20 +19,12 @@ module SwiftWASILibc [system] {
   header "ctype.h"
   header "errno.h"
   header "fenv.h"
-  header "float.h"
```

Comment:
Are these no longer needed or are they transitively included by other headers?

Reply:
They are now part of `_Builtin_XXX` modules provided by clang https://reviews.llvm.org/D159064